README.md

```diff
@@ -136,28 +123,17 @@ cd server
 
 cp src/.mf.config.SAMPLE.json src/.mf.config.json
 
+# To install dependencies, clean/create build/ subdirectory, build and upgrade SDK, build source code within this project
 npm install
-npm run clean             # Cleans/creates build/ subdirectory
 
-npm run build:core        # MANDATORY: If you're only using the Firebolt Core SDK (typical for most 3rd party app devs)
-npm run build:manage      # OPTIONAL: If you're using the Firebolt Manage SDK (not typical)
-npm run build:discovery   # OPTIONAL: If you're using the Firebolt Discovery SDK (not typical)
-
-npm run build:mf          # Builds source code within this project
-
-# Ongoing stuff
-
-# If this is not your first time using mock-firebolt but you want to ensure you have the latest SDK files you can use
-npm run upgradeSDK
-
-# If you're only using the Firebolt Core SDK (typical for most 3rd party app developers)
+# If you want support for Firebolt Core/Manage SDK
 # Run in a separate terminal window/tab, or use '&' to background
 npm run dev
 
-# If you want support for the Firebolt Manage and/or Firebolt Discovery SDKs
+# If you want support for the Firebolt Discovery SDK
 # Include the flag(s) you require
 # Note the extra double dash!
-npm run dev -- --manage --discovery
+npm run dev -- --discovery
```

Comment:
Shouldn't discovery be handled like all other SDKs? If it's in the single open rpc file, it'll be supported automatically / it won't be separate.